textile/build.gradle

```diff
@@ -23,28 +23,46 @@ ext {
 
     licenseName = 'The Apache Software License, Version 2.0'
     licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-    allLicenses = ["Apache-2.0"]
+    allLicenses = ['Apache-2.0']
 }
 
 repositories {
     maven {
-        url "https://dl.bintray.com/textile/maven"
+        url 'https://dl.bintray.com/textile/maven'
     }
 }
 
-android {
-    compileSdkVersion 28
+def getCafeID() {
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file('local.properties').newDataInputStream())
+    return properties.getProperty("cafe.id")
+}
 
+def getCafeToken() {
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file('local.properties').newDataInputStream())
+    return properties.getProperty("cafe.token")
+}
```

Comment:
Better to use env vars, easier with CI. See an example in this file like:
```
String.valueOf(System.getenv("BINTRAY_USERNAME"))
```

Reply:
I tried this for like an hour yesterday... couldn't get it to pick up the system env value.

Reply:
Oh, I see what you're saying... in the gradle file. Will try that, thanks 👊